packages/store-ui/src/Toast/index.tsx

```diff
@@ -4,27 +4,60 @@ import { Box } from 'theme-ui'
 import type { ReactNode, FC } from 'react'
 
 import { CrossIcon } from './CrossIcon'
+import { ErrorIcon } from './ErrorIcon'
+import { WarningIcon } from './WarningIcon'
+import { CheckIcon } from './CheckIcon'
+
+type ToastType = 'warning' | 'error' | 'success'
+
+const typeIcons: Record<ToastType, FC> = {
+  error: ErrorIcon,
+  warning: WarningIcon,
+  success: CheckIcon,
+}
 
 interface Props {
   message: ReactNode
   isVisible: boolean
   hideToast: () => void
+  type: ToastType
```

Comment:
Since we have a default value
```suggestion
  type?: ToastType
```

Reply:
Fixed